geode-core/src/main/java/org/apache/geode/internal/cache/DistributedRegion.java

```diff
@@ -1298,6 +1299,24 @@ private void synchronizeWith(InternalDistributedMember target, VersionSource ver
     op.synchronizeWith(target, versionMember, lostMember);
   }
 
+
+  public void setRegionSynchronizeScheduled(VersionSource lostMemberVersionID) {
```

Comment:
If disk store member is lost more than once, each time it is restarted it will have an initialized false set to the scheduledOrDone flag.
If a member is not lost, the timed task will schedule the region sync method ignoring the scheduledOrDone flag.